### PR TITLE
Issue 755 atom ts view

### LINKED ISSUE #755: Atoms return a view of data (pos/vel/force)
### Expected behaviour

Doing a list comprehension of an Atom Timestep property to get a time series

### Actual behaviour

Get an array of only the first frames data, repeated many times

### Code to reproduce the behaviour

``` python
import MDAnalysis as mda

u = mda.Universe(top, trj)

[u.atoms[0].position for ts in u.trajectory]

```

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)

0.14.1-dev0

Took me a while to get it, but `Atom.position/velocity/force` is returning a view onto the array in Timestep,
so after iteration when the trajectory rewinds, all these views point to the same array address.  Just needs a copy call somewhere

AtomGroup is unaffected as it does a fancy index, and therefore always gets a copy.

### PR DESCRIPTION
**Pull request by richardjgowers**
Fixes #755

Changes made in this Pull Request:
 - Atom properties now return a copy

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by orbeckst** [comment]
Out of curiosity, what is the performance impact of doing the copy?

File: package/MDAnalysis/core/AtomGroup.py [hunk 1]
```diff
@@ -569,7 +569,7 @@ def position(self):
 
         :Returns: a (3,) shape numpy array
         """
-        return self.universe.coord.positions[self.index]  # internal numbering starts at 0
+        return self.universe.coord.positions[self.index].copy()
```
Comment (orbeckst) [inline package/MDAnalysis/core/AtomGroup.py]: Isn't this a fancy index already?

File: package/CHANGELOG [hunk 2]
```diff
@@ -13,12 +13,14 @@ The rules for this file:
...
 ------------------------------------------------------------------------------
-??/??/16 jandom
+??/??/16 jandom, richardjgowers
 
-  * 0.14.1
+  * 0.15.0
 
 Fixes
 
+ * Fixed Atom.position/velocity/force returning a view onto Timestep array
```
Comment (kain88-de) [inline package/CHANGELOG]: Well we return a copy now instead of a view. So just say in the CHANGELOG that we return a copy now for this.

**Comment by kain88-de** [comment]
@richardjgowers what is with residues?

what would  `[u.atoms.residues[0].position for ts in u.trajectory]` give with this PR?

**Comment by kain88-de** [comment]
In general I really like to return copies and make idomatic python code just work. But I'm worried right now that the example you have in the issue will have different results depending on what we use , residue, segment.

**Comment by richardjgowers** [comment]
Ok maybe I've fixed the tests that were failing

**Comment by richardjgowers** [comment]
@MDAnalysis/coredevs this needs reviewing

**Comment by dotsdl** [comment]
@richardjgowers reviewing today

**Comment by dotsdl** [comment]
Looks good!

**Pull request merged**